lib/decision_review/configuration.rb

```diff
@@ -55,5 +55,9 @@ def connection
     def mock_enabled?
       Settings.decision_review.mock || false
     end
+
+    def breakers_error_threshold
+      80
```

Comment:
I think 80 is a percentage, correct? Would you be able to place a comment here? It's not super clear that the integer is actually the percentage of errors over which an outage will be reported

Reply:
Do you actually want to increase the read timeout instead in the above config file? Current configuration: `self.read_timeout = Settings.caseflow.timeout || 20`

Reply:
it is a percentage! yes. i'll note that.

Reply:
After talking this over with @annaswims, I think we wanted to try this route first. Our upstream dependency caches some data, so a second request is often pretty quick, and I think we were nervous about allowing a lot of lengthy connections all at once (Anna correct me if I'm wrong)

Reply:
yeah, the current timeout is set to 40s and we probably don't want to go much higher unless we make it async. Because we're calling lighthouse, there's not much of a chance that lighthouse will be down when we're not.  So breakers is for stats more than for being a circuit breaker.